src/providers/address_info.rs

```diff
@@ -13,4 +13,5 @@ pub struct ContractInfo {
     pub display_name: String,
     pub logo_uri: Option<String>,
     pub contract_abi: Option<Value>,
+    pub trusted_for_delegate_call: Option<bool>,
```

Comment:
I believe, we cache `ContractInfo` for 1 hour (indefinite amount) . To avoid issues similar to what we experienced today, I would feel more comfortable leaving this as `nullable` and opening an issue to move this `non-nullable` after it's safe to do so or we've come up with a migration strategy.